### PR TITLE
Release 0.7 preparations

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -132,30 +139,30 @@ Changelog {#Changelog}
 * [#57](https://github.com/BlueBrain/Livre/pull/57):
   New command line parameter to set the camera lookAt vector to define its
```
Comment (hernando) [inline doc/Changelog.md]: Here there should be another *, shouldn't it?

File: doc/Changelog.md [hunk 2]
```diff
@@ -132,30 +139,30 @@ Changelog {#Changelog}
...
   orientation
-* New command line parameter to set camera position
+  * New command line parameter to set camera position
 * [#17](https://github.com/BlueBrain/Livre/pull/17):
   New command line parameters for RESTBridge (hostname, port, ZeroEQ schema)
 * [#53](https://github.com/BlueBrain/Livre/pull/53):
   New command line parameters to specify the minimum and maximum levels of
```
Comment (hernando) [inline doc/Changelog.md]: same here

**Comment by chevtche** [comment]
Updated

File: doc/Changelog.md [hunk 3]
```diff
@@ -1,29 +1,36 @@
 Changelog {#Changelog}
 =========
 
-# master {#master}
+# Release 0.7 (09-12-2016)
+* [#353](https://github.com/BlueBrain/Livre/pull/353):
+  handle files with right extension and no schema
```
Comment (eile) [inline doc/Changelog.md]: Would you know without context what this means? 

* Allow specification of raw volumes without schema

File: doc/Changelog.md [hunk 4]
```diff
@@ -1,29 +1,36 @@
...
-  Multiple pass mode added for high quality images for data that does not fit into GPU cache
+  Fix missing redraw after expose.
+  * Fix the %300 CPU usage.
+* [#346](https://github.com/BlueBrain/Livre/pull/346):
+  Implement axis and legend for full volume bounding box.
 * [#333](https://github.com/BlueBrain/Livre/pull/333):
-  Add animation-fps parameter to limit the animation frame rate
+  Add animation-fps parameter to limit the animation frame rate.
+* [#331](https://github.com/BlueBrain/Livre/pull/331):
+  Multiple pass mode added for high quality images for data that does not fit into GPU cache.
```
Comment (eile) [inline doc/Changelog.md]: * Added support for rendering high-resolution volumes at full resolution

(the rest are implementation details)

File: doc/Changelog.md [hunk 5]
```diff
@@ -1,29 +1,36 @@
...
 * [#318](https://github.com/BlueBrain/Livre/pull/318):
-  * Added support for NRRD and Raw volumes
+  Added support for NRRD and Raw volumes.
 * [#317](https://github.com/BlueBrain/Livre/pull/317):
-  * Added purge support for cache
-  * The cache objects now uses RAII for allocation of data
-  * Rename TextureDataCache to DataCache, TextureDataObject to DataObject
+  Added purge support for cache.
+  * The cache objects now uses RAII for allocation of data.
+  * Rename TextureDataCache to DataCache, TextureDataObject to DataObject.
```
Comment (eile) [inline doc/Changelog.md]: What does it do for me as a user? The bullet points I definitely don't care about (as a user).

File: doc/Changelog.md [hunk 6]
```diff
@@ -1,29 +1,36 @@
...
 * [#313](https://github.com/BlueBrain/Livre/pull/313):
   Fixed the camera synchronization between applications.
   Replaced the data bounding box by the data to Livre spaces
   transform matrix computed by the datasource for camera
   synchronization.
-* Added meterToDataUnitRatio parameter
-  computed by the datasource.
-* Fixed resolution printing.
+  * Added meterToDataUnitRatio parameter
+    computed by the datasource.
```
Comment (eile) [inline doc/Changelog.md]: What does it do for me as a user?

Comment (chevtche) [inline doc/Changelog.md]: Done

File: doc/Changelog.md (hunk 5 above)
Comment (chevtche) [inline doc/Changelog.md]: Done

File: doc/Changelog.md (hunk 4 above)
Comment (chevtche) [inline doc/Changelog.md]: Done

File: doc/Changelog.md (hunk 3 above)
Comment (chevtche) [inline doc/Changelog.md]: Done

**Comment by chevtche** [comment]
Updated

**Pull request merged**